utils/src/main/java/se/fortnox/reactivewizard/util/ReflectionUtil.java

```diff
@@ -349,4 +306,72 @@ private static Method getMethod(Class<?> cls, String propertyName) {
             return null;
         }
     }
+
+    public static <T> Optional<Supplier<T>> instantiator(Class<T> cls) {
```

Comment:
Unless I'm mistaken, this does pretty much the same as newInstance did but with a more obscure name. Would it be wrong to call it newInstance or createInstance or something more descriptive like that?

Reply:
It does not, it returns a function that can be used to instantiate a class, as opposed to an instance, which was returned from newInstance.